File: contractstudy/src/test/java/contractstudy/collectContracts/cre/UnconditionalOperationNotSupported/java/testData/MultipleUnconditionalOperationNotSupportedException.java
package contractstudy.collectContracts.cre.UnconditionalOperationNotSupported.java.testData;


import java.util.List;

public class MultipleUnconditionalOperationNotSupportedException {

  public void test1(List<String> list) {
    if (list.isEmpty()) {
      throw new UnsupportedOperationException("List is empty");
    }
    throw new UnsupportedOperationException("This is not a JavaCRE");
  }

  public void test2(boolean flag, List<String> list) {
    if (!flag) {
      throw new UnsupportedOperationException("Flag is false");
    }
  }

  public void test3(List<String> list) {
    throw new UnsupportedOperationException("This is not an JavaCRE");
  }
}

File: contractstudy/src/test/java/test/contractstudy/extractor/cre/UnconditionalOperationNotSupported/kotlin/testData/MultipleUnconditionalOperationNotSupportedException.kt
package test.contractstudy.extractor.cre.JavaCREExtractor.kotlin.testData

class MultipleUnconditionalOperationNotSupportedException {
    fun test1(list: List<String?>) {
        if (list.isEmpty()) {
            throw UnsupportedOperationException("List is empty")
        }
        throw UnsupportedOperationException("This is not a JavaCRE")
    }

    fun test2(flag: Boolean, list: List<String?>?) {
        if (!flag) {
            throw UnsupportedOperationException("Flag is false")
        }
    }

    fun test3(list: List<String?>?) {
        throw UnsupportedOperationException("This is not an JavaCRE")
    }
}